packages/forklift-console-plugin/src/modules/Providers/views/create/ProvidersCreatePage.tsx

```diff
@@ -109,6 +113,21 @@ export const ProvidersCreatePage: React.FC<{
           apiError: null,
         };
       }
+      case 'SET_PROJECT_NAME': {
+        const value = action.payload;
+        let validationError: ValidationMsg = { type: 'default' };
+
+        if (!value) {
+          validationError = { type: 'error', msg: 'Missing project name' };
```

Comment:
In case of an empty project list, the field is not enforced as mandatory and the error message is not displayed as required:

![Screenshot from 2024-12-19 14-09-32](https://github.com/user-attachments/assets/67a24e8a-22f8-451f-9f05-c04f986aa23b)